sdk/loganalytics/azure-mgmt-loganalytics/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 0.5.0 (2020-04-22)
+
+**Breaking changes**
+
+  - Change the Client Name back from LogAnalyticsManagementClient
```

Comment:
should be "back to"?
maybe "Reverted client name back to LogAnalyticsManagementClient as previous change was not intentional"

Reply:
okay

Reply:
changed